docs/locking.rst

```diff
@@ -20,11 +20,10 @@ The following example illustrates how to use locking in practice:
     import aerovaldb
 
     with aerovaldb.open('json_files:.') as db:
-        db.acquire_lock()
-        data = db.get_by_uuid('./file.json', default={"counter": 0})
-        data["counter"] += 1
-        db.put_by_uuid(data, './file.json')
-        db.release_lock()
+        async with db.lock():
+            data = db.get_by_uuid('./file.json', default={"counter": 0})
+            data["counter"] += 1
+            db.put_by_uuid(data, './file.json')
```

Comment:
write also an example using 
```
lh = db.lock()
...
lh.release()
```
since people might need to keep the lock open longer than in the current context.